turbo.json

```
@@ -20,17 +20,21 @@
       "outputs": ["dist/**"],
       "dependsOn": ["^build", "compile"]
     },
+    "build:docker": {
+      "inputs": ["Dockerfile"],
```

```suggestion
      "inputs": ["Dockerfile", "./"],
```

So that it will auto rebuild if you change any files in "meta"